lib/iris/fileformats/netcdf/loader.py

```diff
@@ -236,7 +237,19 @@ def _get_cf_var_data(cf_var, filename):
             )
             # Get the chunking specified for the variable : this is either a shape, or
             # maybe the string "contiguous".
+            if CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.AS_DASK:
+                return as_lazy_data(proxy, chunks=None, as_dask=True)
             chunks = cf_var.cf_data.chunking()
+            #
+            if (
+                chunks is None
+                and CHUNK_CONTROL.mode == CHUNK_CONTROL.Modes.FROM_FILE
```

Comment:
```suggestion
                and CHUNK_CONTROL.mode == ChunkControl.Modes.FROM_FILE
```

See my comment about a class variable instead of an instance variable (and sorry for getting this wrong when I was demonstrating!).